### DISCUSSION TITLE
Dexie Cloud Milestone #1: Private Beta

### DISCUSSION THREAD

**Discussion by SuperstringDev** [dexie/Dexie.js#1257] [General]
Recently, I was pleasantly surprised to learn that Dexie Cloud is coming. The app our company is developing is nearing completion and it would be great to use something better than Firebase for data sync and collaboration. 

The app is one that potentially millions of users will engage with and the information presently available about Dexie Cloud sounds like it is a wonderful fit with how we want authentication, access and collaboration to work.

In order to use Dexie Cloud and have it meet our launch schedule we will need to work with the Private Beta. Having just heard about Dexie Cloud I signed up for the Private Beta just this past weekend, before the deadline at end of March. Please let us know what our application status is.

Thank you,
Superstring Media

**Comment by dfahlander** [comment]
Hi! I'm happy to hear you've applied! The application is open until March 31. We'll be going through all new applications on April 1 and let you know!

Kind Regards!
/David
